lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -417,8 +417,8 @@ sub run_backup {
         die $request->{_locale}->text('Error creating backup file');
 
     if ($request->{backup_type} eq 'email') {
-        # suppress warning of single usage of $LedgerSMB::Sysconfig::...
-        no warnings 'once';
```

Comment:
This warning suppression was there to prevent a warning of `$LedgerSMB::Company_Config::settings` being used only once. Without this suppression, the warning will return. The solution is to create a function-based interface.